javascript/ql/src/Vue/ArrowMethodOnVueInstance.qhelp

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+<overview>
+	<p>
+
+		The Vue framework invokes the methods of a Vue instance with the
+		instance as the receiver.  This binding of method and instance is
+		however not possible to do for arrow functions, so the
+		<code>this</code> variable in an arrow function on a Vue instance may
+		not have the value that the programmer expects.
+
+	</p>
+</overview>
+
+<recommendation>
+	<p>
+		Ensure that the methods on a Vue instance can have their receiver bound to the instance.
+	</p>
+</recommendation>
+
+<example>
+
+	<p>
+
+		The following example shows two similar Vue instances, the only
+		difference is how the <code>created</code> life cycle hook
+		callback is defined.
+
+		The first Vue instance uses an arrow function as the callback,
```

Comment:
Sentence a bit too long. I'd suggest splitting it into two: 

The first Vue instance uses an arrow function as the callback. This means that the <code>this</code> variable will have the global object as its value, causing <code>this.myProperty</code> to evaluate to <code>undefined</code>, which may not be intended.